packages/ui-spinner/src/Spinner/index.tsx

```diff
@@ -66,11 +67,8 @@ class Spinner extends Component<SpinnerProps> {
     }
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  constructor(props) {
+  constructor(props: SpinnerProps) {
```

Comment:
Don't forget to add the `@tsProps` flag in the annotations! :)

Also, now that the typing is done, you can remove this block from the `babel.config.js` that removes the unnecessary `@ts-expect-errors`:

```js
  shouldPrintComment(value) {
    const isTsRelatedComment = /@ts-expect-error|FIXME/.test(value)

    return !isTsRelatedComment
  },
```